db/liquibase/changesets/20180915_collection.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="avram" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="avram" author="ansingh" id="collection11">
+        <createTable tableName="COLLECTION">
+            <column name="id" type="BIGINT" autoIncrement="true">
```

Comment:
I think "autoincrement" is a thing in mysql so I looked into this a little. It looks like (though I haven't definitively confirmed) this translates to `SERIAL` in postgres which is basically a macro for a sequence and an integer column with a default value set from the sequence. I don't think there's anything to do here... it's just something you have to know about how liquibase supports postgres.

Reply:
Alternately, I bet we could just use `type="bigserial"` and let postgres do the work instead of liquibase. It depends on whether or not we want to make our liquibase changesets database-agnostic. Since we're planning to use postgres-specific features (for JSON), I think there's more value in embracing postgres and avoiding the indirection of the liquibase abstractions. Therefore, I propose we make it:
```
<column name="id" type="bigserial">
```